### PR TITLE
Add index page for blog

### PR DESCRIPTION
**Pull request by life4ants**

### CODE REVIEW

File: source/blog.html.haml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Blog
+pageable: true
+per_page: 5
+---
+
+%h2.blog_heading
```
Comment (joshukraine) [inline source/blog.html.haml]: Don't use underscores like this in class names. Use hyphens instead. Exception to that would be double underscores for BEM style classes. Syntax example from [this BEM article](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) looks like this:

```css
.site-search {} /* Block */
.site-search__field {} /* Element */
.site-search--full {} /* Modifier */
```

File: source/blog.html.haml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          .meta-author
+            = social_link_for(article.data.author)
+
+        %time.meta-date{ datetime: "#{article.date}" }= pretty_date
+
+      .article-preview__content
+        = article.summary
+        = link_to "Read more", article
+
+  - if paginate && num_pages > 1
```
Comment (joshukraine) [inline source/blog.html.haml]: This shouldn't be here until we have the partial also. We should probably handle pagination in a separate PR.

File: source/blog/articles/2010-05-new-ofr-posted-6.html.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+
+title: "New OFR Posted!"
+date: "2010-05-22 21:17:41"
+tags:
```
Comment (joshukraine) [inline source/blog/articles/2010-05-new-ofr-posted-6.html.markdown]: We're not doing tags right now. All tag references should be removed.

File: source/blog.html.haml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+%h2.blog_heading
+  Euro Team Outreach Blog Articles
+
+%section.narrow_container
+  - page_articles.each_with_index do |article, i|
+    - pretty_date = article.date.strftime("%b %d, %Y")
+
+    %article.article-preview
+      .article-preview__title
+        %h3
```
Comment (joshukraine) [inline source/blog.html.haml]: I think these should be `h2` elements. That's what you're targeting in your css. Right now, the spacing is way off.

https://www.dropbox.com/s/a531gcvapp1vdia/Screenshot%202017-08-17%2019.13.18.png?dl=0

File: source/assets/stylesheets/partials/_blog.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: center;
+}
+
+.narrow_container {
+  max-width: 700px;
+  margin: 50px auto 0 auto;
+}
+
+.article-preview {
+  border-bottom: 1px solid $base-border-color;
```
Comment (joshukraine) [inline source/assets/stylesheets/partials/_blog.scss]: These borders look too dark. Let's use the color used on the [Doctrine page](https://euroteamoutreach.org/doctrine/).

**Comment by life4ants** [comment]
These commits should address all your comments

**Pull request merged**